mantidproject/mantid
# Add SwapWidths algorithm

Fixes #14669 

SwapWidths algorithm added to Mantid
# To Test
- Run the following script ensuring that a path to `IN16B_125878_QLd_Result.nxs` is in your data search directories. It's available from UnitTest and DocTest data (External Data)

```
input_ws = Load(Filename='IN16B_125878_QLd_Result.nxs')
result_ws = SwapWidths(InputWorkspace=input_ws, SwapPoint=5)
```
- Plot f2.f1.FWHM and f2.f2.FWHM from `input_ws` and ensure it looks like the following:
  ![image](https://cloud.githubusercontent.com/assets/13132128/11555952/a2956866-999b-11e5-8e27-9254e8706db0.png)
- Ensure the plotted result looks like the following:
  ![image](https://cloud.githubusercontent.com/assets/13132128/11555905/5cb41770-999b-11e5-8107-f26eb7ada879.png)
- Ensure unit and doc tests both pass
